db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_04_20_165704) do
+ActiveRecord::Schema.define(version: 2023_04_29_100703) do
```

Comment:
**suggestion** just flagging I don't think we need to update the schema here if we take approach noted above